### ISSUE TITLE
FEProblem::forceOutput() vs FEProblem::outputStep(EXEC_FORCED)

### ISSUE THREAD

**Issue by friedmud** [idaholab/moose#5352]
It seems that we have two (slightly different) ways of forcing output.

BOTH are actually used in `TransientMultiApp`!!

We need to consolidate these...

**Comment by aeslaughter** [comment]
FEProblem::forceOutput() should be removed, this was the old way. I can look into this, but probably not today.

**Comment by friedmud** [comment]
Yeah: no problem.  I figured.  Just putting this here so it can be done eventually ;-)

**Comment by aeslaughter** [comment]
I am wrong, we need both.
`forceOutput()` indicates that the next call to outputStep() will happen, this was needed for some reason in the MultiApps system to make sure that an app outputs.

**Comment by permcody** [comment]
Perhaps it's just a naming problem then, does `forceOutput()` only set a flag? If so, maybe it could be renamed to `setForcedOutput()` or something to indicate setting a flag instead of the current verb based name.
